main/SQLHelper.cpp

```diff
@@ -5972,9 +6021,29 @@ void CSQLHelper::UpdateTemperatureLog()
 			{
 			case pTypeRego6XXTemp:
 			case pTypeTEMP:
-			case pTypeThermostat:
 				temp = static_cast<float>(atof(splitresults[0].c_str()));
 				break;
+			case pTypeThermostat:
+				{
+
+					//if not a virtual thermostat
```

Comment:
better check if hardware type is the virtual thermostat, you never know in the feature each thermostat has options

Reply:
ok! check hardware type